templates/python/custom-copilot-rag-customize/src/prompts/chat/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "schema": 1.1,
+  "description": "Chat with Teams RAG",
+  "type": "completion",
+  "completion": {
+      "completion_type": "chat",
+      "include_history": true,
+      "include_input": true,
+      "max_input_tokens": 4096,
+      "max_tokens": 1000,
+      "temperature": 0.9,
+      "top_p": 0.0,
+      "presence_penalty": 0.6,
+      "frequency_penalty": 0.0,
+      "stop_sequences": []
+  },
+  "augmentation": {
+      "augmentation_type": "none",
+      "data_sources": {
+        "local-ai-search": 1200
```

Comment:
Do we want to give the data source another name? This template is not related to Azure AI search.

Reply:
sure, will rename it